modules/nw-networkpolicy-about.adoc

```diff
@@ -123,3 +123,53 @@ spec:
 `NetworkPolicy` objects are additive, which means you can combine multiple `NetworkPolicy` objects together to satisfy complex network requirements.
 
 For example, for the `NetworkPolicy` objects defined in previous samples, you can define both `allow-same-namespace` and `allow-http-and-https` policies within the same project. Thus allowing the pods with the label `role=frontend`, to accept any connection allowed by each policy. That is, connections on any port from pods in the same namespace, and connections on ports `80` and `443` from pods in any namespace.
+
+[id="nw-networkpolicy-allow-from-router_{context}"]
+== allow from router
```

Comment:
```suggestion
== Using the allow-from-router network policy
```

Reply:
This is a tough one because of the name of the policy. But I tried to make suggestions that keep the technical language in tact while limiting allow as a verb: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#a

Reply:
Yes. Was puzzled whether to keep it as it is or to change the heading. Since it was already used in the previous sections. I followed the same.